44.md

```diff
@@ -0,0 +1,169 @@
+NIP-44
+======
+
+Encrypted Payloads (Versioned)
+------------------------------
+
+`optional` `author:paulmillr` `author:staab`
+
+The NIP introduces a versioned encryption data model, allowing multiple algorithm choices to exist simultaneously.
```

Comment:
I would suggest introducing the topic a little better, and then providing some clarity about where and how this should be used.

```suggestion
Nostr can use private key to encrypt data for a particular user (pubkey). Then, that user can decrypt the data with their own private key. This NIP introduces a versioned JSON format for symmetrically encrypted data, allowing multiple algorithm choices to exist simultaneously.

The resulting JSON data could be used in the content of an event, or in a tag. This NIP defines only the data model and supported algorithms and not the events themselves.
```

Reply:
I don't think we need to go quite back to basics, but I've tweaked the wording based on your suggestions. Also, this is generally for asymmetric rather than symmetric encryption.